README.md

```diff
@@ -461,9 +461,8 @@ ANSWERS.addComponent('SearchBar', {
   customHooks: {
     // Optional, a callback invoked when the clear search button is clicked
     onClearSearch: function() {},
-    // Optional, a function invoked when a search is submitted via this component. The users search
-    // is passed in as a string
-    onSubmit: function(searchTerms) {}
+    // Optional, a function invoked when a search is conducted. The search terms are passed in as a string
+    onConductSearch: function(searchTerms) {}
```

Comment:
This is probably why the wording was changed, but it's OK that this is called when we do a defaultInitialSearch in `ANSWERS.init` right? It's wanted behavior that this custom hook in the SearchBar component is called when that happens?

Reply:
Yes exactly! It should be called whenever a search is conducted - default initial search, ANSWERS.search or a search from typing in the search input!